manager/integration/tests/common.py

```diff
@@ -4052,3 +4114,111 @@ def get_engine_image_status_value(client, ei_name):
 def update_setting(client, name, value):
     setting = client.by_id_setting(name)
     client.update(setting, value=value)
+
+
+def create_recurring_jobs(client, recurring_jobs):
+    for name, spec in recurring_jobs.items():
+        client.create_recurring_job(Name=name,
+                                    Task=spec["task"],
+                                    Groups=spec["groups"],
+                                    Cron=spec["cron"],
+                                    Retain=spec["retain"],
+                                    Concurrency=spec["concurrency"])
+        policy = client.by_id_recurring_job(name)
+        assert policy.name == name
+        assert policy.task == spec["task"]
+        assert policy.groups == spec["groups"]
+        assert policy.cron == spec["cron"]
+        assert policy.retain == spec["retain"]
+        assert policy.concurrency == spec["concurrency"]
```

Comment:
Assert here could be optional in common libs but usually used in test cases condition check instead.

Reply:
Make sense. Splited the checking to another function.